pkg/api/admin_users.go

```diff
@@ -29,10 +32,14 @@ func AdminCreateUser(c *models.ReqContext, form dtos.AdminCreateUserForm) Respon
 	}
 
 	if err := bus.Dispatch(&cmd); err != nil {
-		if err == models.ErrOrgNotFound {
+		if errors.Is(err, models.ErrOrgNotFound) {
 			return Error(400, models.ErrOrgNotFound.Error(), nil)
```

Comment:
Do you mind updating this as well? I don't see any point in fixating the error message, when we can get it from `err`:
```suggestion
			return Error(400, err.Error(), nil)
```

Reply:
Well, I'd say it depends. With the current implementation, it's true that both statements are equivalent. However, the usage of `errors.Is` is more recommended than the equality comparator precisely to get it working despite of eventual error wraps. So, with the same reasoning, I'm not sure if returning the message present on given error is what we really want.

Reply:
Personally I think it makes sense to return any wrapping error message, since it should wrap the error for a reason. The wrapped error message should also include `models.ErrOrgNotFound.Error()`. @sakjur @papagian WDYT?

Reply:
I'm pro changing to `err.Error()`.

The only reason I can see why we wouldn't want to use `err.Error()` is to prevent exposure of secrets or over-technical error messages, and since this is an operation only available to admins I don't think that should be a concern.

Alternatively info log `err.Error()` and then return `models.ErrOrgNotFound.Error()`, that way we're not throwing away possibly useful debug information (Does the org not exist? Did the database crash during querying? Timeout while looking?)

Reply:
I agree with @aknuds1; I think we don't want to loose this information.

Reply:
@sakjur I guess that is also my one uncertainty, divulging secrets. My current thinking is that an error wrapping ErrOrgNotFound shouldn't divulge secrets, since it's meant for translating into an API error. That's my thinking around API error design at least, to have certain error types which are meant to be passed on via the API. I'm open to opposing points of view, though. I'm not super used to Go wrapped errors yet, coming from an exception background.

Reply:
Okay, I will change it to `err.Error()`.

Thanks for your point of view!

Reply:
@aknuds1 I agree! If we want to send sensitive/debugging information with the error we should probably extend the error type to have something else than `.Error()` to access that information.

Reply:
That's a good point @sakjur! It's definitely worth thinking about.